### PR TITLE
First pass at PokerBot learnings

### PR DESCRIPTION
**Pull request by GraemeBradbury**
We've decided on doing PokerBot as learning constraint. Initial learning will be around Akka.Net

### CODE REVIEW

**Comment by GraemeBradbury** [comment]
@out-of-band-club What does everyone think of this

Plan is to summarize this into the meeting agenda. With 25th Aug being the earliest we can do, due toe shutdown.

**Comment by GraemeBradbury** [comment]
Okay it seems using org alone then no-one gets notifications.

So thoughts? @vijay-patel @thomasmichaelwallace

File: PokerBot/readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Poker Bot #
+Overall goal with this is to start with the actor system.
+
+We will start with building a system that represents a poker table for playing [limit texas hold'em](https://en.wikipedia.org/wiki/Texas_hold_%27em#Betting_structures)
+
+So for initial meetup I feel we should be aiming to do initial design sketch up and chat about Akka.Net actor model.
+
+I figure everyone should be able to go through the Akka.Net bootcamp before hand so that we have a decent baseline for the discussion.
+[Akka.Net Bootcamp Repo](https://github.com/petabridge/akka-bootcamp)
```
Comment (thomasmichaelwallace) [inline PokerBot/readme.md]: Yup- I'm about mid-way though this :)

Comment (vijay-patel) [inline PokerBot/readme.md]: Yep, sounds good.  About to roll my sleeves up and dig in

**Pull request merged**